source/configuration.txt

```diff
@@ -204,4 +204,4 @@ MongoDB 3.4 added Snappy compression support, while zlib compression was added
 in 3.6, and zstd compression in 4.2.
 
 Data compression can be enabled with the appropriate URI options, as documented
```

Comment:
```suggestion
You can enable data compression the appropriate URI options. Learn about these options
```

Reply:
yeah, left the text alone because this page is currently getting redone as part of standardization. can fix though